src/include/sof/drivers/dmic.h

```diff
@@ -250,6 +252,7 @@
 #define OUTCONTROL1_IPM_SOURCE_2(x)		SET_BITS(12, 11, x)
 #define OUTCONTROL1_IPM_SOURCE_3(x)		SET_BITS(10, 9, x)
 #define OUTCONTROL1_IPM_SOURCE_4(x)		SET_BITS(8, 7, x)
+#define OUTCONTROL1_IPM_SOURCE_MODE(x)	SET_BIT(6, x)
```

Comment:
Can you check here alignment is good (and above) as GH sometimes does not render it correctly for review.

Reply:
When setting the tab to a size of 4 spaces is aligned with the line above.

Reply:
ah, ok - we are using tab size 8. Fwiw, SOF uses the same coding style as Linux and Zephyr. See https://docs.zephyrproject.org/latest/contribute/index.html